Balloon/app/memstat.cpp

```diff
@@ -202,8 +205,25 @@ BOOL CMemStat::Update()
         m_Stats[idx].val = statex.ullTotalPhys;
         idx++;
 
+        status = memory->Get(
+                             L"CacheBytes",
+                             0,
+                             &var_val,
+                             NULL,
+                             NULL
+                             );
+
+        if (FAILED(status) || (var_val.vt == VT_NULL)) {
+            PrintMessage("Cannot get CacheBytes");
+            var_val.vt = 0;
```

Comment:
Why?
Did you miss CWebObject::Get call in the middle, which calls _VariantInit internally?